### PR TITLE
[UPDATE]web_readonly_bypass:migrate from 9 to 10

### PR DESCRIPTION
**Pull request by blutecsolutions**
1.Repair Travis.
2.Make this to work.

### CODE REVIEW

**Comment by blutecsolutions** [comment]
@pedrobaeza 

Can we continue using this PR

**Comment by pedrobaeza** [comment]
But is this working at all?

**Comment by blutecsolutions** [comment]
Changes will be updated ASAP. 👍

File: web_readonly_bypass/__manifest__.py [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
+# Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
 {
     'name': 'Read Only ByPass',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
-    "maintainer": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "maintainer": "ACSONE SA/NV,Odoo Community Association (OCA),Kalpana Hemnani,Blutec Solutions,Kamil Muhammad",
```
Comment (pedrobaeza) [inline web_readonly_bypass/__manifest__.py]: You can't put 3 co-authors here, and less if the changes are so minimal.

Comment (blutecsolutions) [reply web_readonly_bypass/__manifest__.py]: Then we can leave it at two as JS changes will be updated

Comment (pedrobaeza) [reply web_readonly_bypass/__manifest__.py]: No, it should be only one author, and it the changes deserve to be here. The place for putting all the involved persons is in README.rst file, in the section contributors. There you can put several persons if they have worked on that.

Comment (blutecsolutions) [reply web_readonly_bypass/__manifest__.py]: Okay :)

**Comment by yvaucher** [comment]
This PR seems incomplete, I'm closing it in favor of the other most recent ones #743 and #673

**Pull request closed without merging**